packages/core/test/hotkeys/hotkeysTests.tsx

```diff
@@ -114,19 +114,32 @@ describe("Hotkeys", () => {
         });
 
         it("triggers hotkey dialog with \"?\"", (done) => {
+            // this test takes awhile. the hotkey dialog is supposed to take 100ms
```

Comment:
ugh we really want to avoid unavoidably long tests like this, it just slows the whole thing down.

a better approach would be to assert that props are what you'd expect via the enzyme wrapper, rather than waiting for the DOM to catch up.